docker-compose.yml

```diff
@@ -414,13 +419,21 @@ services:
       - RADVIS_WFS_PASSWORD=${RADVIS_WFS_PASSWORD:?missing/empty}
       # Roadworks download url
       - ROADWORKS_SVZBW_DATEX2_DOWNLOAD_URL=${ROADWORKS_SVZBW_DATEX2_DOWNLOAD_URL:?missing/empty}
+      # Webcam settings
+      - IPL_WEBCAM_SERVER=${WEBCAM_FTP_SERVER:?missing/empty}
```

Comment:
[nitpick]

Unless there's a good reason, I'd prefer having the same naming in `.env` and within the container(s).

(The same applies to the variables below.)

Reply:
I would actually prefer it this way: in context of the env var, the IPL prefix does not make much sense, in context of the container, it does.